app/imports/ui/pages/CreateTestimony.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import '/client/style.css';
+import { Col, Container, Image, Row } from 'react-bootstrap';
+import { PAGE_IDS } from '../utilities/PageIDs';
+
+const CreateTestimony = () => (
+  <Container id={PAGE_IDS.CREATETESTIMONY} className="py-3">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous" />
```

Comment:
What does this link do? looks like its trying to use stylesheet or something, but also i dont think crossorigin is a proper class or anything unless you needed to import a dependency to make this work